tutorials/ab3TracingCode.md

```diff
@@ -111,7 +142,17 @@ Recall from the User Guide that the `edit` command has the format: `edit INDEX [
 1. We are interested in the `logic.execute(commandText)` portion of that line so let’s _Step in_ into that method call:<br>
     ![StepInto](images/tracing/StepInto.png)
 
-1. We end up in `LogicManager#execute()` (not `Logic#execute` -- but this is expected because we know the `execute()` method in the `Logic` interface is actually implemented by the `LogicManager` class). Let’s take a look at the body of the method. Given below is the same code, with additional explanatory comments.
+### 2. LogicManager -> AddressBookParser
+
+<annotate src="images/tracing/LogicSequenceDiagramWithMainWindow.png" width="900" alt="Sample Image">
+  <!-- Set Legend to both -->
+  <a-point x="47.5%" y="23%"  label="2"/>
+</annotate>
+
+
+1. After stepping in, we note that `MainWindow` has passed control to the `Logic` component.
+
+1. Specifically, We end up in `LogicManager#execute()` (not `Logic#execute` -- but this is expected because we know the `execute()` method in the `Logic` interface is actually implemented by the `LogicManager` class). Let’s take a look at the body of the method. Given below is the same code, with additional explanatory comments.
```

Comment:
Perhaps include `(not Logic#execute -- but this is expected because we know...)` as a sub-point for better readability? Then followed by `Given below is the same code...`

Reply:
Thanks!, will split into the two points